src/mixins/date-mixin.ts

```diff
@@ -46,12 +46,15 @@ export default class DateMixin extends Mixins(CommonMixin) {
   createUtcDate (year: number, month: number, day: number, hours: number = 0, minutes: number = 0): Date {
     // use date from server to create a new date in Pacific timezone
     // (this sets the correct tz offset in the new date)
```

Comment:
Please delete these obsolete comments.

If you can add another helpful comment explaining what this does (or why it works), that would be great. Or, paste a reference to a SO response.

Reply:
Maybe this?
```
    // 1. create the new date in UTC
    // 2. compute the offset between UTC and Vancouver
    // 3. add the offset to convert the date to Vancouver
```